trunk/SUAVE/Methods/Aerodynamics/Common/Fidelity_Zero/Drag/induced_drag_aircraft.py

```diff
@@ -48,25 +48,23 @@ def induced_drag_aircraft(state,settings,geometry):
     conditions    = state.conditions
     configuration = settings
     
+    K       = configuration.viscous_lift_dependent_drag_factor
+    CDp     = state.conditions.aerodynamics.drag_breakdown.parasite.total
+    CL      = conditions.aerodynamics.lift_coefficient
+    CDi_inv = conditions.aerodynamics.drag_breakdown.induced.total
+    e_inv   = geometry.wings['main_wing'].span_efficiency     

```

Comment:
Not for this PR, but I think this needs to go.

Reply:
Agreed